runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -401,8 +407,9 @@ JITServerAOTCache::getClassLoaderRecord(const uint8_t *name, size_t nameLength)
    if (it != _classLoaderMap.end())
       return it->second;
 
-   auto record = AOTCacheClassLoaderRecord::create(_classLoaderMap.size() + 1, name, nameLength);
+   auto record = AOTCacheClassLoaderRecord::create(_nextClassLoaderId, name, nameLength);
```

Comment:
Nitpick: Instead of incrementing `_nextClassLoaderId` on a separate line, it could have been done in:
`auto record = AOTCacheClassLoaderRecord::create(_nextClassLoaderId++, name, nameLength);`

Reply:
`AOTCacheClassLoaderRecord::create()` and `addToMap()` throw exceptions if they fail to allocate memory. I think it's better to create new IDs only for successfully added records.